docs/courseInstance.md

```diff
@@ -72,37 +70,47 @@ The course instance `allowAccess` rules determine who can access the course inst
 
 ```json title="infoCourseInstance.json"
 {
-  "groupAssessmentsBy": "Module"
+  "allowAccess": [
+    {
+      "startDate": "2015-01-19T00:00:01",
+      "endDate": "2015-05-13T23:59:59",
+      "institution": "Any"
+    }
+  ]
 }
 ```
 
-For more information about assessment modules, see [Course configuration](course/index.md#assessment-modules).
+### Controlling access by UID
 
-## Timezone
-
-The default timezone for course instances is the timezone of the course. This can be changed with the `timezone` property in `infoCourseInstance.json`. For example:
+You can restrict access to particular students by listing their UIDs in the `uids` property. For instance, the following rule only allows `student1@example.com` and `student2@example.com` to access the course instance between the specified dates:
 
 ```json title="infoCourseInstance.json"
 {
-  "timezone": "America/New_York"
+  "allowAccess": [
+    {
+      "startDate": "2015-01-19T00:00:01",
+      "endDate": "2015-05-13T23:59:59",
+      "uids": ["student1@example.com", "student2@example.com"]
+    }
+  ]
 }
 ```
 
-Allowable timezones are those in the TZ column in the [list of tz database time zones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones), which is a display version of the [IANA Time Zone Database](https://www.iana.org/time-zones).
-
 ## Enrollment controls
 
 Students can enroll in a course instance through one of two ways:
 
-1. They can use a URL specific to the course instance or [to one of its assessments](assessment/index.md#linking-to-assessments). The appropriate link to provide to students can be found by opening the "Settings" tab of the Course Instance. This page includes, among other useful information, a Student Link that can be provided to students. This link points students to the list of assessments associated to the course instance, enrolling them automatically in the course instance if they are not yet enrolled.
+1. They can use a URL specific to the course instance or [to one of its assessments](assessment/index.md#linking-to-assessments). You can find the "student link" on the "Settings" tab of the course instance. This link points students to the list of assessments associated to the course instance, enrolling them automatically in the course instance if they are not yet enrolled.
 
-2. They can use the "Enroll" button in PrairieLearn's main page. This button opens a page listing all course instances that are currently available for enrollment, giving students the option to Add new courses.
+2. They can use the "Add or remove courses" button on PrairieLearn's homepage. This button opens a page listing all course instances that are currently available for enrollment, giving students the option to add new courses.
```

Comment:
There's also the following button, right? Should we change this one to also be "Add or remove courses" for consistency?
<img width="646" alt="image" src="https://github.com/user-attachments/assets/486629a8-7b83-44a7-9912-25c3a8b1119c" />

Reply:
Good catch, I agree we want this to be consistent. Done in 6a39d65 (didn't see the need to split this into a separate PR).